Frontend/ui-driver/src/App.purs

```diff
@@ -15,47 +15,46 @@
 
 module Types.App where
 
-import Control.Transformers.Back.Trans (BackT)
 import Control.Monad.Except.Trans (ExceptT)
 import Control.Monad.Free (Free)
```

Comment:
How all the imports are being changed? Can you not use prettier or any other extension to sort imports. Doing this will cause issues for other devs and introduce redundant  code